docs/rdt.md

```diff
@@ -48,5 +48,5 @@ CRI-RM has a built-in default configuration containing three classes
 corresponding to the Pod QOS classes of Kubernetes. These are utilized by the
 `static` policy.
 
-See `rdt` in the [example ConfigMap spec](../sample-configs/cri-resmgr-configmap.example.yaml)
+See `rdt` in the [example ConfigMap spec](/sample-configs/cri-resmgr-configmap.example.yaml)
```

Comment:
It would have been nicer to distill this (and other similar fixes to the raw docs) into a separate 'docs: documentation fixes.'-commit. But it's fine like this now, no need to split it out any more.